src/main/java/com/supermartijn642/movingelevators/mixin/MovingElevatorsMixinPlugin.java

```diff
@@ -15,12 +15,10 @@
  */
 public class MovingElevatorsMixinPlugin implements IMixinConfigPlugin {
 
-    private boolean isSodiumLoaded;
     private boolean isIrisLoaded;
 
     @Override
     public void onLoad(String mixinPackage){
-        this.isSodiumLoaded = isClassAvailable("me.jellysquid.mods.sodium.client.SodiumClientMod");
         this.isIrisLoaded = isClassAvailable("net.coderbot.iris.Iris");
```

Comment:
`net.irisshaders.iris.Iris`?

Reply:
I think I'm just going to remove this check entirely and replace it with a `@Pseudo`.